defaults/main.yml

```diff
@@ -3,9 +3,6 @@
 # ansible-role-django-k8s performs a two-step deploy process.
 # First, it uses your (admin) credentials to create a namespace and service account;
 # those admin credentials should be defined with a combination of these variables:
-# (i.e., in your local ~/.kube/config):
-k8s_kube_config: "~/.kube/config"
-k8s_kube_context: ""
```

Comment:
That's a good point. I like the idea of something in version control that would nail down which cluster we are supposed to be deploying to.

I've updated this to use ``k8s_auth_host`` for that. It'll require ``k8s_auth_host`` to have been set already, and it'll specify that as the host on all k8s tasks to make sure we're using that cluster. How does that look?

Reply:
I think this makes sense. Since the approach is different (basically the opposite) from what #13 was suggesting, it took me a bit to wrap my head around it. I was struggling with:

* I'm currently deploying to multiple clusters quite often, which I could achieve previously with a simple ``ansible-playbook -l my-cluster`` limit and it would pick up the correct ``k8s_kube_context`` automatically. 
* Additionally, we don't currently have ``k8s_auth_api_key`` committed to the repo anywhere. In fact, one of our repos isn't using ``ansible-vault`` at all. CI uses an environment variable and I didn't need it locally using my ``k8s_kube_context``.

So at first I felt like requiring the user to have the current context active and ``k8s_auth_api_key`` set was a drawback, but I see the value of requiring use of the deploy account. Perhaps #13 should just be closed.

I haven't used Digitial Ocean K8s yet, but I've noticed use of ``k8s_kube_config`` in conjunction with DO, like [here](https://github.com/caktus/caktus-hosting-services/blob/master/Ansible/services/inventory/host_vars/inddex-cluster.yaml#L4). I don't know why this is. I assume the new configuration will still work? Maybe @tobiasmcnulty or @vkurup can provide insight.